src/parser.pegjs

```diff
@@ -1,7 +1,114 @@
 {
+  var parse = require('./parse');
+  var utils = require('./utils');
+  var conj = utils.conj;
 }
 
 
-start = dialogue
+start
+  = ws* dialogue:dialogue ws*
+  { return dialogue }
 
-dialogue = .*
+
+dialogue 'dialogue'
+  = title:dialogueTitle ws* properties:properties? ws* sequences:sequences?
+  {
+    return conj(properties || {}, {
+      title: title,
+      sequences: sequences || []
+    });
+  }
+
+
+dialogueTitle 'dialogue title'
+  = '#' lineWs* value:text
+  { return value; }
+
+
+sequenceTitle 'sequence title'
+  = '##' lineWs* value:text
+  { return value; }
+
+
+blockTitle 'block title'
+  = '###' lineWs* value:text
+  { return value; }
+
+
+sequences 'sequences'
+  = (s:sequence ws* { return s; })+
+
+
+sequence 'sequence'
+  = title:sequenceTitle ws* properties:properties? ws* blocks:blocks?
+  {
+    return conj(properties || {}, {
+      title: title,
+      blocks: blocks || []
+    });
+  }
+
+
+blocks 'blocks'
+  = (b:block ws* { return b; })+
+
+
+block 'block'
+  = title:blockTitle ws* properties:properties?
+  { return conj(properties || {}, {title: title}); }
+
+
+properties 'properties'
+  = first:property rest:(newline ws* p:property { return p; })*
```

Comment:
If looks like this doesn't allow whitespace at the end of a property line? Not sure if this is a feature or a bug. :)